bluesky_kafka/__init__.py

```diff
@@ -141,7 +145,18 @@ def __call__(self, name, doc):
             event-model document dictionary
 
         """
-        self.produce(message=(name, doc))
+        try:
+            self.produce(message=(name, doc))
+        except KafkaException as e:
+            KErr = e.args[0]
+            if KErr.code() == KafkaError.MSG_SIZE_TOO_LARGE:
+                if name == "event_page":
+                    page_len = len(doc['seq_num'])
```

Comment:
```suggestion
                    page_len = len(doc['seq_num'])
                    if page_len == 1:
                        raise
```

I think in the case of a pathologically wide page we would end up in an infinite recursion.

Reply:
It's hard to imagine ever getting there, but I'm glad you imagined it. Recursion without a base case is dangerous!